src/ghost_story_factory/orchestration/graph.py

```
@@ -26,6 +32,9 @@ def create_story_graph() -> StateGraph:
     Returns:
         编译后的 StateGraph
     """
+    if not _LANGGRAPH_AVAILABLE:
+        raise RuntimeError("LangGraph 不可用，请安装 langgraph 或关闭 USE_LANGGRAPH_PIPELINE")
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)